docs/dapp/tools/localnet.mdx

```diff
@@ -0,0 +1,161 @@
+ * Waiting for key manager......
+ * Populating accounts...
+
+Available Accounts
+==================
+(0) 0x41b0C13e747F8Cb1c4E980712504437cb1792327 (10000 TEST)
+(1) 0xa521f94f8a38b1d027D526017EB229327B9D6cA0 (10000 TEST)
+(2) 0x1e0f8369215D6C5Af5E14eD6A0D6ae7372776A79 (10000 TEST)
+(3) 0xB60cA28B491747a27C057AdBF3E71F3CCC52332C (10000 TEST)
+(4) 0x88D7d924e521a6d07008a373D5b33281148ffEDc (10000 TEST)
+
+Private Keys
+==================
+(0) 0x617346c545d62b8213ea907acf1b570a7405683e2c6dcaf963fc21fd677e0c56
+(1) 0xf82d6e09208b0bd44a397f7e73b05c564e6c9f70b151ee7677e2bb8d6ce5d882
+(2) 0xeb2f21d20086f3dd6bfe7184dad1cb8b0fb802f27b1334e836a19eda0a43a1c2
+(3) 0x82b0203d6063992b1052004b90411c45d4f3afab696346f006e74c6abd8f855e
+(4) 0x7179c6e1add3a2993822653b9c98fe606f47fb6d4c0d0d81b31b067cf6bb5f83
+
+HD Wallet
+==================
+Mnemonic:       coach genre beach child crunch champion tell adult critic peace canoe stable
+Base HD Path:   m/44'/60'/0'/0/%d
+
+WARNING: The chain is running in ephemeral mode. State will be lost after restart!
+
+ * Listening on http://localhost:8545 and ws://localhost:8546. Chain ID: 0x5afd
+ * Container start-up took 66 seconds, node log level is set to warn.
+```
+
+Those familiar with local dApp environments will find the output above similar
+to `geth --dev` or `ganache-cli` commands or the `geth-dev-assistant` npm
+package. The [sapphire-localnet] will spin up a private Oasis Network locally,
+generate and populate test accounts and make the following Web3 endpoints
+available for you to use:
+
+- `http://localhost:8545`
+- `ws://localhost:8546`
+
+:::tip
+
+If you prefer using the same mnemonics each time (e.g. for testing purposes)
+or to populate just a single account, use `-to` flag and pass the mnemonics or
+the wallet addresses. By passing the `-test-mnemonic` flag you can fund the
+standard test accounts provided by the `hardhat node` commmand and that are
+typically used for solidity unit tests.
+
+<Tabs>
+<TabItem value="Sapphire">
+
+```sh
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -to "bench remain brave curve frozen verify dream margin alarm world repair innocent" -n3
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -to "0x75eCF0d4496C2f10e4e9aF3D4d174576Ee9010E2,0xbDA5747bFD65F08deb54cb465eB87D40e51B197E"
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -test-mnemonic
+```
+
+</TabItem>
+<TabItem value="Emerald">
+
+```sh
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -to "bench remain brave curve frozen verify dream margin alarm world repair innocent" -n3
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -to "0x75eCF0d4496C2f10e4e9aF3D4d174576Ee9010E2,0xbDA5747bFD65F08deb54cb465eB87D40e51B197E"
+docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -test-mnemonic
+```
+
+</TabItem>
+</Tabs>
+
+:::
```

Comment:
The `-test-mnemonic` is now default. Let's keep this outside of admonition not to bloat the chapter too much.

````suggestion
By default, the Localnet docker image will populate the first five accounts
derived from the standard test mnemonic, compatible with `hardhat node`.
These accounts are typically used for solidity unit tests. If you prefer
populating different accounts, use `-to` flag and pass the mnemonics or the
wallet addresses. Using the `-n` parameter defines the number of derived
addresses to fund.

<Tabs>
<TabItem value="Sapphire">

```sh
docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -to "bench remain brave curve frozen verify dream margin alarm world repair innocent" -n3
docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/sapphire-localnet -to "0x75eCF0d4496C2f10e4e9aF3D4d174576Ee9010E2,0xbDA5747bFD65F08deb54cb465eB87D40e51B197E"
```

</TabItem>
<TabItem value="Emerald">

```sh
docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -to "bench remain brave curve frozen verify dream margin alarm world repair innocent" -n3
docker run -it -p8544-8546:8544-8546 ghcr.io/oasisprotocol/emerald-localnet -to "0x75eCF0d4496C2f10e4e9aF3D4d174576Ee9010E2,0xbDA5747bFD65F08deb54cb465eB87D40e51B197E"
```

</TabItem>
</Tabs>
````